wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/environment/server/compose/EnvironmentContextTest.java

```diff
@@ -0,0 +1,131 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.environment.server.compose;
+
+import com.google.common.collect.ImmutableMap;
+import org.eclipse.che.api.core.ServerException;
+import org.testng.annotations.Test;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.testng.Assert.assertEquals;
+
+/**
+ * @author Dmytro Nochevnov
+ */
+public class EnvironmentContextTest {
+
+    ComposeFileParser parser = new ComposeFileParser();
+
+    @Test
+    public void shouldParseDictionaryTypeEnvironment() throws ServerException {
```

Comment:
Please add tests for next cases:
- map format, value of variable is empty
- map format, value of variable contains colon sign
- map format, no colon in entry
- array format, no equal sign in entry
- array format, empty value of variable
- array format, value of variable contains equal sign